source/support/support-overview.rst

```diff
@@ -0,0 +1,179 @@
+
+      You can register for SUBNET from the MinIO Console.
+
+      #. Go to your MinIO cluster's URL, then sign in
+      #. Select the :guilabel:`Support` option
+      #. Select :guilabel:`Register`
+      #. Enter your :guilabel:`Email` address and a unique :guilabel:`Password`, then select :guilabel:`Register`
+
+      For clusters without direct Internet access:
+
+      #. Go to your MinIO cluster's URL, then sign in
+      #. Select the :guilabel:`Support` option
+      #. Select :guilabel:`Register`
+      #. Select the :guilabel:`Offline Activation` link at the bottom of the screen
+      #. Copy the registration token for your cluster from item 1
+      #. Go to https://subnet.min.io/cluster/register and enter the registration token when prompted
+      #. Copy the generated API key and paste it in item 3 of the **Offline Registration** window in the MinIO Console
+      #. Select :guilabel:`Register`
+
+   .. tab-item:: Command Line
+
+      You can register for SUBNET from the command line.
+
+      Refer to :mc-cmd:`mc support register` for instructions.
+
+
+SUBNET Issues
+-------------
+
+Use SUBNET issues to engage support from MinIO engineering.
+
+#. Log in to https://subnet.min.io
+#. Select the :guilabel:`Issues` section
+
+Use the search bar to locate an existing issue or add a new issue.
+
+.. image:: /images/subnet/issues-section.png
+   :width: 600px
+   :alt: MinIO SUBNET with the Issues section displaying a list of an organization's issues
+   :align: center
+
+Select an existing issue from the list to expand the converation or add a response.
+
+.. image:: /images/subnet/issue-expanded.png
+   :width: 600px
+   :alt: A example MinIO SUBNET issue conversation
+   :align: center
+
+Reviewing Health Data in SUBNET
+-------------------------------
+
+SUBNET provides health data about the clusters registered to the organization from the :guilabel:`Deployments` section.
+
+The view shows the total size of the org's MinIO clusters with details for each cluster.
+
+.. image:: /images/subnet/deployments-overview.png
+   :width: 600px
+   :alt: MinIO SUBNET displaying the deployments overview
+   :align: center
+
+Each of the organization's clusters display below the summary data.
+Select a deployment row to view additional health details.
+
+Deployment Health
+~~~~~~~~~~~~~~~~~
+
+The deployment's details include a summary of the deployment's configuration and the number of checks run and failed.
+You can select :guilabel:`Upload` to add diagnostic health data obtained from the :mc-cmd:`mc support diagnostics` command or the MinIO Console's Support > Health page.
+
+If you need support from MinIO Engineering, you can create a :guilabel:`New Issue` for the deployment.
+
+.. image:: /images/subnet/SUBNET-deployment-health-summary.png
+   :width: 600px
+   :alt: MinIO SUBNET displaying health summary information for a myminio deployment
+   :align: center
+
+When available, the following checks display for the deployment:
```

Comment:
Thought - we have to maintain this list as it changes if we specify it above. I'm of a mind to just make this generic like "SUBNET displays health checks for data points such as CPU, drives, memory, network, and security."